### ISSUE TITLE
WCAG 2.1 Accessibility Criteria

### ISSUE THREAD

**Issue by ibocevska** [atutor/ATutor#161]
Does ATutor comply with the following Success Criteria? Thanks in advance.

Success Criterion 2.1.4 Character Key Shortcuts (Level A): If a keyboard shortcut is implemented in content using only letter (including upper- and lower-case letters), punctuation, number, or symbol characters, then at least one of the following is true: 
Turn off
A mechanism is available to turn the shortcut off; 
Remap
A mechanism is available to remap the shortcut to use one or more non-printable keyboard characters (e.g. Ctrl, Alt, etc); 
Active only on focus
The keyboard shortcut for a user interface component is only active when that component has focus. 

Success Criterion 1.4.12 Text Spacing (Level AA): In content implemented using markup languages that support the following text style properties, no loss of content or functionality occurs by setting all of the following and by changing no other style property: 
•	Line height (line spacing) to at least 1.5 times the font size;
•	Spacing following paragraphs to at least 2 times the font size;
•	Letter spacing (tracking) to at least 0.12 times the font size;
•	Word spacing to at least 0.16 times the font size.

**Comment by atutor** [comment]
Spacing would depend on the theme being used. Easily adjusted if it doesn't meet your specs.

As for shortcuts, probably not compliant. WCAG 2.1 - 2.1.4 came after ATutor's the last development cycle. ATutor uses HTML accesskeys.
